frame/balances/src/lib.rs

```diff
@@ -161,6 +161,7 @@ mod tests_local;
 #[cfg(test)]
 mod tests_reentrancy;
 pub mod weights;
+pub mod migration;
```

Comment:
What would get migrated here? If it counts reserved balances for example, it could end up double counting depending on where it's used, like with crowdloans.

Reply:
As per the PR's summary:

```
Balances pallet provides a migration which can be used to mark accounts with inactive funds (which, after the upgrade, we assume will be kept up to date with deactivate and reactivate). For chains with XCM, this should be used to mark the Checking Account when upgraded.
```